packages/calcite-design-tokens/src/tokens/core/size.json

```diff
@@ -276,6 +276,13 @@
           "attributes": {
             "category": "size"
           }
+        },
+        "none": {
```

Comment:
Should this be `normal` (to match https://github.com/Esri/calcite-design-system/pull/12828/files#diff-7937085cf48fe3ccde0548979da213232c5269b8577173a75a74546fe0613347R280)?